airbyte-integrations/connectors/source-s3/source_s3/source_files_abstract/formats/json_parser.py

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+from typing import Any, BinaryIO, Iterator, Mapping, TextIO, Union
+
+from pandas.io.json import build_table_schema, read_json
+
+from .abstract_file_parser import AbstractFileParser
+
+# All possible json data types
+JSON_TYPES = {
+    # Table Schema type: (json_type, convert_function)
+    # standard types
+    "string": ("string", None),
+    "boolean": ("boolean", None),
+    "number": ("number", None),
+    "integer": ("integer", None),
+    # supported by Pandas type
+    "datetime": ("string", lambda v: v.isoformat()),
+    "object": ("string", lambda v: v.isoformat()),
+    "duration": ("string", lambda v: v.isoformat()),
+    "any": ("string", lambda v: v.isoformat()),
```

Comment:
Are you sure this type support iso format conversion 🤔 ? 
```suggestion
    "object": ("string", lambda v: v.isoformat()),
    "duration": ("string", lambda v: v.isoformat()),
    "any": ("string", lambda v: v.isoformat()),
```

Reply:
Thanks for pointing that out. I realized I'm not actually calling this function. I may have to refactor some stuff to take it into account. I'll check the conversion methods at the same time.